SLE-854: Fix JDT with COBOL

thahnen opened the pull request:
When having the Java Development Tools (JDT) installed but working with COBOL files with non-Java projects, we have to make sure JDT does not try to see it as a 'Java-like' file due to the extension (e.g., `cbl`). Otherwise, it would try to exclude it from the analysis because it is not in the Java classpath and, therefore, not compiled.

JDT itself tries to find 'Java-like' file extensions by the standard Java file suffixes and from other ones it seems. I've created an [issue](https://github.com/eclipse-jdt/eclipse.jdt.core/issues/2389) with Eclipse JDT just in case this is actually a bug.

jblievremont commented on its/src/org/sonarlint/eclipse/its/StandaloneAnalysisTest.java:
```diff
@@ -128,6 +129,19 @@ public void analyze_automatic_workspace_build_disabled() {
+    openFileAndWaitForAnalysisCompletion(cobolFile);
+
+    assertThat(new SonarLintConsole().getConsoleView().getConsoleText())
+      .doesNotContain("File 'Test.cbl' excluded by 'JavaProjectConfiguratorExtension'");
```
Is there a way to make sure that the file was actually analyzed, at least by the secrets analyzer?

thahnen replied:
Done, the test is changed to test for both.

jblievremont reviewed (approved):
LGTM from a code point of view, I would love to see a "positive" assertion in the test.

thahnen committed:
SLE-854: Fix JDT with COBOL

When having JDT installed but working with COBOL files with non-Java projects, we have to make sure JDT does not try to see it as 'Java-like' file due to the extension (e.g. cbl).
